src/composables/useFormatDate.ts

```diff
@@ -0,0 +1,72 @@
+// libraries
+import { date } from 'quasar';
+
+// composables
+import { i18n } from 'src/boot/i18n';
+
+export const useFormatDate = () => {
```

Comment:
For better code readibility split this long function to small sub functions, please.

E.g.

This func should be standalone function and etc.:

```ts
const formatDateTimeLabel = (dateString: string): string => {
    const timeStamp = new Date(dateString);
    const nowStamp = new Date();

    if (!date.isValid(dateString)) {
      return '';
    }
    const isPast = timeStamp.getTime() < nowStamp.getTime();
    if (!isPast) {
      return '';
    }
```

Reply:
Refactored into a function determining a state (using ternary operators) and a switch based on the state.
Check for `date.isValid(dateString)` (line 34) needs to use `if` statements for early return.